doc/classes/OS.xml

```diff
@@ -224,18 +224,21 @@
 			<return type="int">
 			</return>
 			<description>
+				Return the amount of battery left in the device as a percentage.
 			</description>
 		</method>
 		<method name="get_power_seconds_left">
 			<return type="int">
 			</return>
 			<description>
+				Return the time in seconds before the device runs out of battery.
 			</description>
 		</method>
 		<method name="get_power_state">
 			<return type="int" enum="OS.PowerState">
 			</return>
 			<description>
+				Return the current state of the device regarding battery and power. Indicated in terms of [OS.PowerState].
```

Comment:
Return -> Returns

Per writing guidelines, you should change the last sentence to:

    See [code]POWERSTATE_*[/code] constants.